staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -48,6 +48,14 @@ const (
 	DiskErrorPolicyEnospace DiskErrorPolicy = "enospace"
 )
 
+type PanicDeviceModel string
+
+const (
+	Hyperv  PanicDeviceModel = "hyperv"
+	Isa     PanicDeviceModel = "isa"
+	Pvpanic PanicDeviceModel = "pvpanic"
```

Comment:
> Are we sure that we need to support all of these?

I'm not sure about the need for `isa` and `pvpanic`. I added them since they're supported by libvirt.

> Also, how are we going to treat s390?

s390 doesn't need any support. libvirt automatically adds panic devices for s390 (See https://libvirt.org/formatdomain.html#id112)

Reply:
What happens when a user mistakenly sets the panic device to a value that is not supported by s390, do we need any kind of validation?

Reply:
Do you mean validations to not allow panic devices in the VMI spec when its architecture is s390x?

Reply:
@vladikr Can you clarify your question? In general, qemu/libvirt will reject unsupported panic devices in the domxml.

Reply:
We should validate possible values per architecture

Reply:
I've added a validation to not allow panic devices for s390 and p-series architectures.